python/opencc/__init__.py

```diff
@@ -39,15 +39,23 @@
 
 _opencc_share_dir = os.path.join(_thisdir, 'clib', 'share', 'opencc')
 CONFIGS = [
-    os.path.splitext(f)[0]
-    for f in os.listdir(_opencc_share_dir)
+    f for f in os.listdir(_opencc_share_dir)
     if f.endswith('.json')
 ]
 
 
 class OpenCC(object):
 
     def __init__(self, config='t2s.json'):
+        if not os.path.isfile(config):
+            if not config.endswith('.json'):
+                config += '.json'
```

Comment:
I don't feel this necessary. C++ API does not support adding '.json' if absent.

Reply:
Resolving due to outdated